kolibri/core/assets/test/views/connection-snackbars-spec.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-env mocha */
+import Vue from 'vue-test';
+import { mount } from 'avoriaz';
+import assert from 'assert';
+import Vuex from 'vuex';
+import sinon from 'sinon';
+import coreSnackbar from 'kolibri.coreVue.components.coreSnackbar';
+import connectionSnackbars from '../../src/views/connection-snackbars';
+import {
+  tryToReconnect,
+  showDisconnectedSnackbar,
+  showTryingToReconnectSnackbar,
+  showSuccessfullyReconnectedSnackbar,
+} from '../../src/state/actions';
+
+const createStore = () =>
+  new Vuex.Store({
+    state: {
+      core: {
+        connection: {
+          connected: true,
+          reconnectTime: null,
+        },
+        currentSnackbar: null,
+      },
+    },
+    mutations: {
+      CORE_SET_CURRENT_SNACKBAR(state, snackbar) {
```

Comment:
Flagging that a desirable refactor would be to move the mutations out of core/store.js so they could be imported into the test instead of re-implementing it here.

A bonus would be to implement store.js as a Factory, basically serving as `createStore` for tests as well as app code.

Reply:
Yeah, I think that would help to reduce the overhead in our tests, and increase confidence that our tests are reflective of production.